src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.ts

```diff
@@ -40,14 +49,22 @@ export class NotificationPopupComponent implements OnInit {
      * @param notification {Notification}
      */
     navigateToTarget(notification: Notification): void {
-        this.router.navigateByUrl(this.notificationTargetRoute(notification));
+        // tslint:disable-next-line:triple-equals
+        if (notification.title == LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE) {
+            const target = JSON.parse(notification.target!);
+            this.examExercisesNavigationService.navigateToExamExercise(target.exercise);
+        } else {
+            this.router.navigateByUrl(this.notificationTargetRoute(notification));
+        }
     }
 
     private notificationTargetRoute(notification: Notification): UrlTree | string {
         if (notification.target) {
             const target = JSON.parse(notification.target);
-
-            if (notification.title === 'Quiz started' && target.status) {
+            // tslint:disable-next-line:triple-equals
+            if (notification.title == LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION_TITLE) {
```

Comment:
Maybe you can rethink if you really want to do all the "matching" for your specific case via the title. I guess it could make sense to introduce a dedicated case (also in the backend) for your scenario, i.e., instead of adding an if-statement in the `EXERCISE_UPDATE` case in `src/main/java/de/tum/in/www1/artemis/domain/notification/GroupNotificationFactory.java`, add a new case.

Reply:
This is a very difficult decision that I already discussed with Stephan but we did not find a solution last time. 

I already tried adding a new notification type (this seems to be the go-to way), but sadly this information about the new type gets lost due to the creation of the group notification. 
Group notification extends from the normal notifications but swaps the types with new ones. These new ones should not be extended for this notification-related cause IMO.  
GroupNotifications have the following types: INSTRUCTOR, EDITOR, TA, STUDENT
A LIVE_EXAM_EXERCISE_UPDATE_NOTIFICATION does not seem right to me.

I could try using single-user-notifications but this would be (as I think) very strange if you look at the already established rules and logic that is used in the notification system (e.g. quiz workflow, etc.).

Group Notifications already provide a very solid and (more or less) easily adjustable structure.

My first idea was to add a new notification Priority "EXAM" but Stephan told me this does not fit with the other priorities (low, medium, high).
So I need a constant value that stays the same throughout the notification's journey & smth that is in line with the legacy code -> I think the title is the best fit for this. 
If you or @krusche think otherwise please say so but please also provide a logic of how to make it work
because I already tried multiple different options and encountered (major) drawbacks, or was simply stuck.
E.g. it worked with query params but Stephan told me that the URLs should not be changes so I looked for other options and this one is IMO the best one (so far).